src/lib/check-in.ts

```diff
@@ -18,45 +92,37 @@ export function makeFetchCheckinDataAttempts(
     };
   };
 
-  return SwClient.loadJsonPage<Response>({
-    url: SwClient.withSuffix(
-      'mobile-air-operations/v1/mobile-air-operations/page/check-in/',
-      reservation
-    ),
-    method: SwClient.Method.GET,
-    headers,
-    retry: {
-      // We'll try a total of 80 times.
-      // This should give us 5 seconds of checkin tries before checkin time and 15 seconds after.
-      limit: 79,
-      methods: [SwClient.Method.POST],
```

Comment:
Should there be a POST for the other "new" way? Maybe not, as it seems like the checkin happens in two steps. First get the data on `'mobile-air-operations/v1/mobile-air-operations/page/check-in/'`, then post to another url to complete the checkin.

Reply:
Yeah I was looking at that too, but I think this`methods: [SwClient.Method.POST]` line was unnecessary before.

We were issuing a GET request in this function before and we're still issuing a GET request.